AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -217,6 +217,11 @@
 
     private static final String MARK_CARD = "markCard";
     private static final String TOGGLE_FLAG = "toggleFlag";
+    private static final String BURY_CARD = "buryCard";
+    private static final String TAG_CARD = "tagCard";
```

Comment:
I think the symbols here could be very specific as there may be lots of future tag work, so specifying the full verb/subject/object tuple is useful 

```suggestion
    private static final String ADD_TAG_TO_CARD = "addTagToCard";
```